client/integration-tests/path/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.rest-client.quarkus_simple_openapi_yaml.url=http://localhost:8080
+
+quarkus.package.jar.decompiler.enabled=true
```

Comment:
Why do we need this property?

Reply:
Sorry, we don't need that. I will correct that.